### ISSUE TITLE
Extension issue

### ISSUE THREAD

**Issue by r0manr0k0n** [hoovercj/vscode-power-mode#62]
- Issue Type: `Bug`
- Extension Name: `vscode-power-mode`
- Extension Version: `2.2.0`
- OS Version: `Linux x64 5.0.0-37-generic`
- VSCode version: `1.41.1`

```json 
{
	"messages": [],
	"activationTimes": {
		"codeLoadingTime": 1,
		"activateCallTime": 2,
		"activateResolvedTime": 264,
		"activationReason": {
			"startup": true,
			"extensionId": {
				"value": "imagio.vscode-dimmer-block",
				"_lower": "imagio.vscode-dimmer-block"
			},
			"activationEvent": "*"
		}
	},
	"runtimeErrors": [
		{
			"name": "TypeError",
			"message": "Cannot read property 'document' of undefined"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "Error",
			"message": "Illegal argument: line must be non-negative"
		},
		{
			"name": "TypeError",
			"message": "Cannot read property 'document' of undefined"
		},
		{
			"name": "TypeError",
			"message": "Cannot read property 'document' of undefined"
		},
		{
			"name": "TypeError",
			"message": "Cannot read property 'document' of undefined"
		},
		{
			"name": "TypeError",
			"message": "Cannot read property 'document' of undefined"
		},
		{
			"name": "TypeError",
			"message": "Cannot read property 'document' of undefined"
		},
		{
			"name": "TypeError",
			"message": "Cannot read property 'document' of undefined"
		}
	]
}
```

**Comment by hoovercj** [comment]
@rokonxr I see that the error mentions "imagio.vscode-dimmer-block". Is there a reason that you believe the issue is with power mode and not with vscode-dimmer-block?

**Comment by rokonxr** [comment]
@hoovercj I don't know. VS Code created the issue, see below image...
![Screenshot from 2020-05-02 20-24-56](https://user-images.githubusercontent.com/36927037/80866973-72b94300-8cb3-11ea-84e8-a879c465a3d3.png)

**Comment by rokonxr** [comment]
Now there's a new error message: 

```json 
{
	"messages": [],
	"activationTimes": {
		"codeLoadingTime": 15,
		"activateCallTime": 2,
		"activateResolvedTime": 159,
		"activationReason": {
			"startup": true,
			"extensionId": {
				"value": "hoovercj.vscode-power-mode",
				"_lower": "hoovercj.vscode-power-mode"
			},
			"activationEvent": "*"
		}
	},
	"runtimeErrors": [
		{
			"name": "TypeError",
			"message": "Cannot read property 'selection' of undefined"
		},
		{
			"name": "TypeError",
			"message": "Cannot read property 'selection' of undefined"
		}
	]
}
```

**Comment by heroboy** [comment]
here is the stack trace of second error.
https://github.com/hoovercj/vscode-power-mode/issues/51

**Comment by hoovercj** [comment]
Thanks again for reporting this! I believe I fixed anything that could have caused this in the just-shipped v3.0
